decode/data.ts

```diff
@@ -30,8 +30,13 @@ export function decode(tokens: Token[]): DataEvent {
             return { time, event, data: tag };
         case Event.Target:
             let targetData: TargetData[] = [];
-            for (let t = 2; t < tokens.length; t += 3) {
-                let target: TargetData = { id: tokens[t] as number, hash: tokens[t + 1] as string, box: tokens[t + 2] as number[] };
+            for (let t = 2; t < tokens.length; t += 4) {
+                let target: TargetData = {
+                    id: tokens[t] as number,
+                    hash: tokens[t + 1] as string,
+                    region: tokens[t + 2] as string,
```

Comment:
>region [](start = 20, length = 6)

What does this represent?

Reply:
A region on the page that may include multiple elements. This is similar to box model selector concept.